Source/WebKit/WebProcess/InjectedBundle/API/gtk/DOM/WebKitDOMKeyboardEvent.h

```diff
@@ -105,13 +105,12 @@ webkit_dom_keyboard_event_get_modifier_state(WebKitDOMKeyboardEvent* self, const
  * @altKey: A #gboolean
  * @shiftKey: A #gboolean
  * @metaKey: A #gboolean
- * @altGraphKey: A #gboolean
  *
  *
  * Deprecated: 2.22: Use JavaScriptCore API instead
 **/
 WEBKIT_DEPRECATED void
-webkit_dom_keyboard_event_init_keyboard_event(WebKitDOMKeyboardEvent* self, const gchar* type, gboolean canBubble, gboolean cancelable, WebKitDOMDOMWindow* view, const gchar* keyIdentifier, gulong location, gboolean ctrlKey, gboolean altKey, gboolean shiftKey, gboolean metaKey, gboolean altGraphKey);
+webkit_dom_keyboard_event_init_keyboard_event(WebKitDOMKeyboardEvent* self, const gchar* type, gboolean canBubble, gboolean cancelable, WebKitDOMDOMWindow* view, const gchar* keyIdentifier, gulong location, gboolean ctrlKey, gboolean altKey, gboolean shiftKey, gboolean metaKey);
```

Comment:
This is public API and you cannot change it. Instead, keep the parameter and add a call to `UNUSED_PARAM(metaKey)` to document that it's intentionally no longer used (and avoid tripping `-Wunused-parameter`).

Reply:
Actually sorry, as Chris has mentioned in https://github.com/WebKit/WebKit/pull/20404#discussion_r1462564906 we should of course just omit the parameter name from the function signature in the source file rather than use `UNUSED_PARAM()` here.

Reply:
OK - so just one change of `false` to `FALSE`?

Reply:
No, you need to keep the `altGraphKey` parameter. That cannot be removed because it is public.